utils/src/server/rpc_module.rs

```diff
@@ -169,12 +163,6 @@ impl MethodCallback {
 				let tx = tx.clone();
 				let params = params.into_owned();
 				let id = id.into_owned();
-				tracing::trace!(
-					"[MethodCallback::execute] Executing async callback, params={:?}, req.id={:?}, conn_id={:?}",
-					params,
-					id,
-					conn_id
-				);
```

Comment:
How come you want these gone?

Reply:
those were a bit noisy because we both print `execute_with_resources` and `execute` but I can revert those

Reply:
Your call, I don't mind either way.